guacamole/src/main/frontend/src/translations/cs.json

```diff
@@ -123,8 +124,10 @@
         "HELP_MOUSE_MODE_RELATIVE" : "Přetažením myši posuňte ukazatel myši a klepněte na tlačítko. Kliknutí nastane v místě ukazatele.",
         "HELP_SHARE_LINK"          : "Aktuální připojení je sdíleno a může k němu přistupovat kdokoli s následujícím {LINKS, plural, one{odkazem} other{odkazy}}:",
 
-        "INFO_CONNECTION_SHARED" : "Toto připojení je nyní sdíleno.",
-        "INFO_NO_FILE_TRANSFERS" : "Žádné přenosy souborů.",
+        "INFO_ANONYMOUS_USER_COUNT" : "Anonymní{COUNT, plural, one{} other{ (#)}}",
+        "INFO_CONNECTION_SHARED"    : "Toto připojení je nyní sdíleno.",
+        "INFO_NO_FILE_TRANSFERS"    : "Žádné přenosy souborů.",
+        "INFO_USER_COUNT"           : "{USERNAME}{COUNT, plural, one{} other{ (#)}}",
```

Comment:
I think you can leave this line out, `INFO_USER_COUNT`, as there is no actual translation going on, here.